google/cloud/security/common/data_access/sql_queries/snapshot_cycles_sql.py

```diff
@@ -24,6 +24,7 @@
         `complete_time` datetime DEFAULT NULL,
         `status` enum('SUCCESS','RUNNING','FAILURE',
                       'PARTIAL_SUCCESS','TIMEOUT') DEFAULT NULL,
+        `has_all_data` bool DEFAULT NULL,
```

Comment:
@blueandgold I leave it up to you to know if the other enum statuses are being used and if moving to a boolean is satisfactory given the ways a pipeline can fail.